src/lib/globals.ts

```diff
@@ -11,6 +12,19 @@ declare global {
     var _azleGuardFunctions: { [key: string]: () => any };
 }
 
+if (globalThis._azleIc) {
+    globalThis.console = {
+        ...globalThis.console,
+        log: (...args: any[]) => {
+            const jsonStringifiedArgs = args
+                .map((arg) => JSON.stringify(arg, replacer, 4))
+                .join(', ');
```

Comment:
Both node and chrome don't join with a `,`, they just join with a space...

Reply:
Fixed